### ISSUE TITLE
Plot in octave

### ISSUE THREAD

**Issue by Jjhayi** [ibell/coolprop#241]
![captura de tela 2014-10-02 15 52 10](https://cloud.githubusercontent.com/assets/9000464/4496507/06ca38c4-4a66-11e4-8733-1ac2aca643a2.png)
Hi.
I can't plot a graphic in octave if i use Coolprop.
The error that appears is in the image
Any help is welcome :)

Sorry about my bad english

**Comment by ibell** [comment]
Can you attach the entire script please? I don't think this is a CoolProp problem.

**Comment by Jjhayi** [comment]
But if i don't start the "Coolprop.oct" i can plot normally

With coolprop
![captura de tela 2014-10-02 16 13 55](https://cloud.githubusercontent.com/assets/9000464/4496681/5559fbb2-4a68-11e4-9bd5-c0c889146e12.png)

Without coolprop
![captura de tela 2014-10-02 16 16 00](https://cloud.githubusercontent.com/assets/9000464/4496722/a281032c-4a68-11e4-8b4b-c3bf66079b41.png)

PS:By script you mean the entire error? Because that's all that appears

**Comment by ibell** [comment]
That is very weird.  What is at line 28 in your script file?

**Comment by ibell** [comment]
If you are not too invested into octave, I really recommend python for
these sorts of things.  Your script would only be a few lines in python.
Especially on windows the octave support is quite bad.

Ian

**Comment by Jjhayi** [comment]
just plot(x,Tc,"b",x,Th,"r")

Date: Thu, 2 Oct 2014 12:20:10 -0700

**Comment by ibell** [comment]
hmm.  How can you plot without having coolprop loaded?

**Comment by Jjhayi** [comment]
Using the same thing.

**Comment by Jjhayi** [comment]
If i use coolprop, should i plot in a different way?

**Comment by ibell** [comment]
It shouldn't have any impact on plotting at all.  Seems like a bug in
Octave.

**Comment by Jjhayi** [comment]
well, i gonna try to reinstall 
if don't work, i gonna try to learn python :)

**Comment by ibell** [comment]
Yes I think it is a good investment to learn python.  It works great for
these sorts of problems.  It is my primary language, so if you have
problems I can guarantee we can fix them.  Octave is pretty buggy.

**Comment by Jjhayi** [comment]
Okey dokey sir :)
Thanks for your time
